docs/tutorials/docker-compose.yml

```diff
@@ -2,160 +2,49 @@
 version: '2'
 services:
   zookeeper:
-    image: "confluentinc/cp-zookeeper:4.1.0"
-    hostname: zookeeper
-    ports:
-      - '32181:32181'
+    image: "confluentinc/cp-zookeeper:5.0.0-beta180702222458"
     environment:
       ZOOKEEPER_CLIENT_PORT: 32181
       ZOOKEEPER_TICK_TIME: 2000
-    extra_hosts:
-      - "moby:127.0.0.1"
 
   kafka:
-    image: "confluentinc/cp-enterprise-kafka:4.1.0"
-    hostname: kafka
+    image: "confluentinc/cp-enterprise-kafka:5.0.0-beta180702222458"
     ports:
-      - '9092:9092'
-      - '29092:29092'
+      - '39092:39092'
     depends_on:
       - zookeeper
     environment:
       KAFKA_BROKER_ID: 1
       KAFKA_ZOOKEEPER_CONNECT: zookeeper:32181
       KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
       KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:29092,PLAINTEXT_HOST://localhost:9092
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:39092,PLAINTEXT_HOST://localhost:9092
       KAFKA_AUTO_CREATE_TOPICS_ENABLE: "true"
       KAFKA_METRIC_REPORTERS: io.confluent.metrics.reporter.ConfluentMetricsReporter
       KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
-      CONFLUENT_METRICS_REPORTER_BOOTSTRAP_SERVERS: kafka:29092
+      KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS: 100
+      CONFLUENT_METRICS_REPORTER_BOOTSTRAP_SERVERS: kafka:39092
       CONFLUENT_METRICS_REPORTER_ZOOKEEPER_CONNECT: zookeeper:32181
       CONFLUENT_METRICS_REPORTER_TOPIC_REPLICAS: 1
       CONFLUENT_METRICS_ENABLE: 'true'
       CONFLUENT_SUPPORT_CUSTOMER_ID: 'anonymous'
-    extra_hosts:
-      - "moby:127.0.0.1"
 
   schema-registry:
-    image: "confluentinc/cp-schema-registry:4.1.0"
-    hostname: schema-registry
+    image: "confluentinc/cp-schema-registry:5.0.0-beta180702222458"
     depends_on:
       - zookeeper
       - kafka
-    ports:
-      - '8081:8081'
     environment:
       SCHEMA_REGISTRY_HOST_NAME: schema-registry
       SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL: zookeeper:32181
-    extra_hosts:
-      - "moby:127.0.0.1"
-
-  # Runs the Kafka KSQL data generator for topic called "pageviews"
-  ksql-datagen-pageviews:
```

Comment:
why are we getting rid of this stuff?

Reply:
Datagen is now run inline as part of the quickstart.